docs/community/pr.md

```diff
@@ -4,7 +4,9 @@
 
 `PR` is the abbreviation of `Pull Request`. Here's the definition of `PR` in the [official document](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) of Github.
 
-> Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch.
+```
+Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch.
+```
```

Comment:
![image](https://user-images.githubusercontent.com/76149310/145187754-2a68055f-9de2-4739-9099-26098a31568f.png)
a little weird, I thought it was caused by `'` in `you've`

Reply:
done